java/test/src/main/java/org/ray/api/test/CrossLanguageInvocationTest.java

```diff
@@ -57,6 +57,24 @@ public void testCallingPythonFunction() {
     Assert.assertEquals(res.get(), "Response from Python: hello".getBytes());
   }
 
+  @Test
+  public void testCrossLanguageSerialization() {
```

Comment:
I think existing test use bytes is OK, because bytes is not using msgpack. There is another test case for pure serialization: java/runtime/src/test/java/org/ray/runtime/util/SerializerTest.java
Also, I will complement the serialization test for all supported types.

Reply:
using bytes could work, but it was a workaround and might be confusing to people.

Reply:
OK. I will treat bytes as a primitive type to test serialization.